frontend/src/components/EditTodo.js

```diff
@@ -0,0 +1,78 @@
+import { Fragment, useState } from "react";
+
+const EditTodo = ({todo}) => {
+    
+    const [description, setDescription] = useState(todo.description);
+
+    const updateDescription = async (e) => {
+        e.preventDefault();
+        try {
+            const body = {description};
+            const response = await fetch(`http://localhost:5000/todos/${todo.todo_id}`, {
+                method: "PUT",
+                headers: {"Content-Type": "application/json"},
+                body: JSON.stringify(body)
+            });
+            window.location = "/"
+        } catch (error) {
+            console.log(error.message)
+        }
+    }
+
+    return (
+        <Fragment>
+            
+            <button 
+            type="button" 
+            class="btn btn-warning" 
+            data-toggle="modal" 
+            data-target={`#id${todo.todo_id}`}>
```

Comment:
data-target={`#id${todo.todo_id}`}> in button 
and below
 id={`id${todo.todo_id}`} in div, to ensure that each button gets the description of its todo, default was static id that made each edit having last todo value